playbooks/allocate_pubcloud.yml

```diff
@@ -20,6 +20,11 @@
       retries: 120
       delay: 60
 
+    - name: Fail if the required number of instances aren't available
+      fail:
+        msg: "At least one public cloud instance failed to start :("
+      when: "{{ rax.success|length < count }}"
```

Comment:
Nit, Ansible says to [leave off the double curly brackets][1]:
```yaml
when: rax.success|length < count
```
[1]: http://docs.ansible.com/ansible/playbooks_conditionals.html#the-when-statement